specification/cognitiveservices/OpenAI.Inference/models/audio/commons.tsp

```diff
@@ -31,6 +31,17 @@ Requesting format 'json' will result on only the 'text' field being set.
 For more output data use 'verbose_json.
 """)
 model AudioTranscriptionOptionsCommon {
```

Comment:
Given we're going for accuracy, we should likely add "model" to this and hide it (hide it at least until we incorporate the feedback to get rid of model/deployment as an operation parameter from the customer perspective)